plugins/vpc-shared-eni/network/bridge_windows.go

```diff
@@ -285,7 +285,14 @@ func (nb *BridgeBuilder) DeleteEndpoint(nw *Network, ep *Endpoint) error {
 	log.Infof("Detaching HNS endpoint %s from container %s netns.", hnsEndpoint.Id, ep.ContainerID)
 	err = hcsshim.HotDetachEndpoint(ep.ContainerID, hnsEndpoint.Id)
 	if err != nil {
-		return err
+		if err == hcsshim.ErrComputeSystemDoesNotExist {
```

Comment:
Instead of these nested if block, can we simply do:

```
if err != nil && err != hcsshim.ErrComputeSystemDoesNotExist {
    return err
}
```

Reply:
Yeah the reason I added the extra block is to log the error message to see how often we encounter this error scenario. Sure I can do this, will update the CR.